### PR TITLE
Add openshift stackrox testgrids

### PR DESCRIPTION
**Pull request by gavin-stackrox**
/cc @porridge

### CODE REVIEW

File: config/testgrids/openshift/redhat-stackrox-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      url: https://prow.ci.openshift.org/job-history/<gcs_prefix>
+    test_group_name: branch-ci-stackrox-stackrox-nightlies-style-checks
+  - base_options: width=10
+    name: branch-ci-stackrox-stackrox-nightlies-ui-unit-tests
+    open_test_template:
+      url: https://prow.ci.openshift.org/view/gcs/<gcs_prefix>/<changelist>
+    results_url_template:
+      url: https://prow.ci.openshift.org/job-history/<gcs_prefix>
+    test_group_name: branch-ci-stackrox-stackrox-nightlies-ui-unit-tests
+  name: redhat-stackrox-merge
```
Comment (porridge) [inline config/testgrids/openshift/redhat-stackrox-nightly.yaml:185]:
```suggestion
  name: redhat-stackrox-nightlies
```

**Review by porridge — APPROVED** [review]

**Comment by porridge** [comment]
/lgtm
apart from that one `name:`

**Review by janisz — APPROVED** [review]

**Comment by gavin-stackrox** [comment]
dang. bad YAML. fixed & ran `make unit` locally.
/retest-required

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**